packages/apidom-ast/src/Node.ts

```diff
@@ -1,45 +1,42 @@
-import stampit from 'stampit';
-
 import Position from './Position';
 
-interface Node {
-  type: string | null;
-  isMissing: boolean;
-  children: unknown[];
-  position: Position | null;
+export interface NodeConstructor {
```

Comment:
You don't need to do this anymore.

When you define Node class

**node.ts**
```ts
class Node {
}
export default Node;
```
It defines two interfaces for you under the hood

1. instance interface
2. constructor interface

**Accessing instance interface**
```ts
import Node from './node';

interface Test extends Node {
}
```

**Accessing constructor interface**
```js
import Node from './node';

type NodeConstructor = typeof Node;
```